freemocap-ui/src/layout/Navigator.tsx

```diff
@@ -20,7 +20,7 @@ const categories = [
       },
       {
         id: "Board Detection",
-        route: "/board_detection"
+        route: "/charuco_board_detection"
```

Comment:
when doing the renames, make sure to watch out for changes in the frontend code

Reply:
makes sense, will do. 

I've been using PyCharm's "refactor" functions for all the renaming. I'll keep an eye out for front-end changes in the future. 

If there's some easy way to configure things so that happens automatically, let me know